android/modules/ui/res/drawable/titanium_icon_more.xml

```diff
@@ -3,7 +3,7 @@
     android:height="24dp"
     android:viewportWidth="24.0"
     android:viewportHeight="24.0"
```

Comment:
The other icons were not affected? Not sure actually.

Reply:
The chevron (and other icons) are actually the correct size (they weren't scaled correctly before), but for parity with previous SDKs I've made the chevron a little smaller here.